package.json

```diff
@@ -243,5 +243,6 @@
       "import": "./dist/es/parseAst.js"
     },
     "./dist/*": "./dist/*"
-  }
+  },
+  "packageManager": "pnpm@9.15.2+sha512.93e57b0126f0df74ce6bff29680394c0ba54ec47246b9cf321f0121d8d9bb03f750a705f24edc3c1180853afd7c2c3b94196d0a3d53d3e069d9e2793ef11f321"
```

Comment:
Why is this added here? We do not use pnpm at the moment in this repo.

Reply:
Oh, I'm sorry. The `COREPACK_ENABLE_AUTO_PIN` has misbehaved!

Reply:
@lukastaegert I fixed at here and I'm sorry! 🙏 

6696dcd